Added University of York

iangray001 opened the pull request:
Note:
In Alan Burns' PR, you added the following comment "This is a disambiguation page (it should be Steve King 0001).". 

However, neither Steve's homepage, nor Scholar link, are to disambiguation pages, so I am not sure what this meant.


diff --git a/country-info.csv b/country-info.csv
index 123f8778..8c7b497c 100644
--- a/country-info.csv
+++ b/country-info.csv
@@ -224,3 +224,4 @@ University of Surrey,europe
 University of Vienna,europe
 Fudan University,asia
 Korea University,asia
+University of York,europe
diff --git a/csrankings.csv b/csrankings.csv
index 98b87ebf..e0d98d14 100644
--- a/csrankings.csv
+++ b/csrankings.csv
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
 Mayank Singh 0001,IIT Gandhinagar,http://www.iitgn.ac.in/faculty/comp/mayank.htm,FOjZrQcAAAAJ
 Nipun Batra 0001,IIT Gandhinagar,https://www.iitgn.ac.in/faculty/comp/nipun.htm,rFGzHlIAAAAJ
+Robert I. Davis,University of York,https://www-users.cs.york.ac.uk/~robdavis/,xjVBcr4AAAAJ
+Iain Bate,University of York,https://www-users.cs.york.ac.uk/~ijb/,dE_ooMIAAAAJ
+Leandro Soares Indrusiak,University of York,https://www-users.cs.york.ac.uk/~lsi/,f1PUDsEAAAAJ
+Wanli Chang,University of York,https://www-users.cs.york.ac.uk/~wanli/,OYcT_B4AAAAJ
+Andy J. Wellings,University of York,https://www-users.cs.york.ac.uk/~andy/,yPa4QrkAAAAJ
+Neil C. Audsley,University of York,https://www-users.cs.york.ac.uk/~neil/,D65ZWSgAAAAJ
+Ian Gray,University of York,https://www-users.cs.york.ac.uk/~iang/,HloK4CAAAAAJ
+Alan Burns,University of York,https://www-users.cs.york.ac.uk/~burns/,Io7KbYcAAAAJ
+Ibrahim Habli,University of York,https://www-users.cs.york.ac.uk/~ihabli/,U9j-FS8AAAAJ
+Tangming Yuan,University of York,https://www-users.cs.york.ac.uk/~tommy/,1fzy4fsAAAAJ
+Lilian Blot,University of York,https://www-users.cs.york.ac.uk/~lblot/,Enkx56UAAAAJ
+Radu Calinescu,University of York,https://www-users.cs.york.ac.uk/~raduc/,NLiNqPwAAAAJ
+Daniel Kudenko,University of York,https://www-users.cs.york.ac.uk/~kudenko/,uUXUMukAAAAJ
+Daniel W. Franks,University of York,https://danfranks.weebly.com,Zpo8koYAAAAJ
+Javier Cámara,University of York,http://www.javicamara.com/,iMMncw8AAAAJ
+Tim P. Kelly,University of York,https:/www-users.cs.york.ac.uk/~tpk/,cNeDb_8AAAAJ
+Dimitar Kazakov,University of York,https://www-users.cs.york.ac.uk/~kazakov/,CLI_4LYAAAAJ
+Stefano Pirandola,University of York,https://www-users.cs.york.ac.uk/~pirs/,bV8llfkAAAAJ
+Christopher Power,University of York,https://www-users.cs.york.ac.uk/~cpower/,1ZA0JtcAAAAJ
+Paul Cairns,University of York,https://www-users.cs.york.ac.uk/~pcairns/,cNVmuG0AAAAJ
+Helen Petrie,University of York,https://www-users.cs.york.ac.uk/~petrie/,xhkqoGAAAAAJ
+Suresh Manandhar,University of York,https://www-users.cs.york.ac.uk/~suresh/,5iH8GVIAAAAJ
+Simon O'Keefe,University of York,https://www-users.cs.york.ac.uk/~sok/,QL1JHlUAAAAJ
+Ana Cavalcanti,University of York,https://www-users.cs.york.ac.uk/~alcc/,fu6TM4wAAAAJ
+Siamak F. Shahandashti,University of York,https://www-users.cs.york.ac.uk/~siamak/,CZTajNQAAAAJ
+Susan Stepney,University of York,https://www-users.cs.york.ac.uk/~susan/,rD32w_EAAAAJ
+Edwin R. Hancock,University of York,https://www-users.cs.york.ac.uk/~erh/,EjDU2ncAAAAJ
+Adrian G. Bors,University of York,https://www-users.cs.york.ac.uk/~adrian/,cvdyalUAAAAJ
+Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det/,nh6AHbIAAAAJ
+Nicholas Drivalos Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos/,Dvp7fNoAAAAJ
+John A McDermid,University of York,https://www-users.cs.york.ac.uk/~jam/,REscLkoAAAAJ
+William A. P. Smith,University of York,https://www-users.cs.york.ac.uk/wsmith/,o2z0h6MAAAAJ
+Anders Drachen,University of York,https://www.cs.york.ac.uk/people/?group=Academic%20and%20Teaching%20Staff&username=adrachen,rOcL0NgAAAAJ
+Samuel L. Braunstein,University of York,https://www-users.cs.york.ac.uk/~schmuel/,jh-mlXoAAAAJ
+Angus M. Marshall,University of York,https://www-users.york.ac.uk/~amm616/,RV4F9n8AAAAJ
+Chris Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb/,QQ_dWtcAAAAJ
+Steven A. Wright,University of York,https://www-users.cs.york.ac.uk/~saw/,uh1UaD4AAAAJ
+Dimitrios S. Kolovos,University of York,https://www-users.cs.york.ac.uk/~dkolovos/,aRd1i6UAAAAJ
+James Cussens,University of York,https://www-users.cs.york.ac.uk/~jc/,OEFkOR0AAAAJ
+Steve King,University of York,https://www-users.cs.york.ac.uk/~king/,Q75-BIgAAAAJ
+Rob Alexander,University of York,https://www-users.cs.york.ac.uk/~rda/,DFdBpC0AAAAJ
+Richard C. Wilson,University of York,https://www-users.cs.york.ac.uk/~wilson/,o4Sq5yMAAAAJ
+Nick Pears,University of York,https://www-users.cs.york.ac.uk/nep/,U_hUs6wzw_AC
+Vassilios G. Vassilakis,University of York,https://www-users.cs.york.ac.uk/~vv/,-OzxgUwAAAAJ
+Siyuan Ji,University of York,https://www.cs.york.ac.uk/people/?group=Academic%20and%20Teaching%20Staff&username=ji,oZdrNNwAAAAJ
+Jim Woodcock,University of York,https://www-users.cs.york.ac.uk/~jim/,bO-FeTIAAAAJ
+Peter Nightingale,University of York,https://www-users.cs.york.ac.uk/pwn503/,Msh14AUAAAAJ
+Ioanna Iacovides,University of York,https://www-users.york.ac.uk/~ii560/,LuBaasYAAAAJ
+Richard F. Paige,University of York,https://www-users.cs.york.ac.uk/~paige/,HzFjuyUAAAAJ
+Peter I. Cowling,University of York,https://www-users.cs.york.ac.uk/~pic/,xQ7IpL8AAAAJ
+Delaram Kahrobaei,University of York,https://www-users.cs.york.ac.uk/~delaram/,ScTS9PkAAAAJ
+Jim Austin,University of York,https://www-users.cs.york.ac.uk/~austin/,xpZL8d8AAAAJ
+James Alfred Walker,University of York,https://www.cs.york.ac.uk/people/?group=Research%20Fellows%20and%20Research%20Associates&username=jawalker,Yl5OycsAAAAJ
diff --git a/dblp-aliases.csv b/dblp-aliases.csv
index 6e4ade2e..38c9bbe5 100644
--- a/dblp-aliases.csv
+++ b/dblp-aliases.csv
@@ -32637,4 +32637,6 @@ Torsten Kroeger,Torsten Kröger
 Alexander H. Waibel,Alex Waibel
 Walter Tichy,Walter F. Tichy
 Anne Martens,Anne Koziolek
-David Mimno,David M. Mimno
\ No newline at end of file
+David Mimno,David M. Mimno
+Chris Bailey,Christopher Bailey 0002
+Jim Austin,James Austin

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

iangray001 committed:
Added University of York

diff --git a/country-info.csv b/country-info.csv
index 123f8778..8c7b497c 100644
--- a/country-info.csv
+++ b/country-info.csv
@@ -224,3 +224,4 @@ University of Surrey,europe
 University of Vienna,europe
 Fudan University,asia
 Korea University,asia
+University of York,europe
diff --git a/csrankings.csv b/csrankings.csv
index 98b87ebf..e0d98d14 100644
--- a/csrankings.csv
+++ b/csrankings.csv
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
 Mayank Singh 0001,IIT Gandhinagar,http://www.iitgn.ac.in/faculty/comp/mayank.htm,FOjZrQcAAAAJ
 Nipun Batra 0001,IIT Gandhinagar,https://www.iitgn.ac.in/faculty/comp/nipun.htm,rFGzHlIAAAAJ
+Robert I. Davis,University of York,https://www-users.cs.york.ac.uk/~robdavis/,xjVBcr4AAAAJ
+Iain Bate,University of York,https://www-users.cs.york.ac.uk/~ijb/,dE_ooMIAAAAJ
+Leandro Soares Indrusiak,University of York,https://www-users.cs.york.ac.uk/~lsi/,f1PUDsEAAAAJ
+Wanli Chang,University of York,https://www-users.cs.york.ac.uk/~wanli/,OYcT_B4AAAAJ
+Andy J. Wellings,University of York,https://www-users.cs.york.ac.uk/~andy/,yPa4QrkAAAAJ
+Neil C. Audsley,University of York,https://www-users.cs.york.ac.uk/~neil/,D65ZWSgAAAAJ
+Ian Gray,University of York,https://www-users.cs.york.ac.uk/~iang/,HloK4CAAAAAJ
+Alan Burns,University of York,https://www-users.cs.york.ac.uk/~burns/,Io7KbYcAAAAJ
+Ibrahim Habli,University of York,https://www-users.cs.york.ac.uk/~ihabli/,U9j-FS8AAAAJ
+Tangming Yuan,University of York,https://www-users.cs.york.ac.uk/~tommy/,1fzy4fsAAAAJ
+Lilian Blot,University of York,https://www-users.cs.york.ac.uk/~lblot/,Enkx56UAAAAJ
+Radu Calinescu,University of York,https://www-users.cs.york.ac.uk/~raduc/,NLiNqPwAAAAJ
+Daniel Kudenko,University of York,https://www-users.cs.york.ac.uk/~kudenko/,uUXUMukAAAAJ
+Daniel W. Franks,University of York,https://danfranks.weebly.com,Zpo8koYAAAAJ
+Javier Cámara,University of York,http://www.javicamara.com/,iMMncw8AAAAJ
+Tim P. Kelly,University of York,https:/www-users.cs.york.ac.uk/~tpk/,cNeDb_8AAAAJ
+Dimitar Kazakov,University of York,https://www-users.cs.york.ac.uk/~kazakov/,CLI_4LYAAAAJ
+Stefano Pirandola,University of York,https://www-users.cs.york.ac.uk/~pirs/,bV8llfkAAAAJ
+Christopher Power,University of York,https://www-users.cs.york.ac.uk/~cpower/,1ZA0JtcAAAAJ
+Paul Cairns,University of York,https://www-users.cs.york.ac.uk/~pcairns/,cNVmuG0AAAAJ
+Helen Petrie,University of York,https://www-users.cs.york.ac.uk/~petrie/,xhkqoGAAAAAJ
+Suresh Manandhar,University of York,https://www-users.cs.york.ac.uk/~suresh/,5iH8GVIAAAAJ
+Simon O'Keefe,University of York,https://www-users.cs.york.ac.uk/~sok/,QL1JHlUAAAAJ
+Ana Cavalcanti,University of York,https://www-users.cs.york.ac.uk/~alcc/,fu6TM4wAAAAJ
+Siamak F. Shahandashti,University of York,https://www-users.cs.york.ac.uk/~siamak/,CZTajNQAAAAJ
+Susan Stepney,University of York,https://www-users.cs.york.ac.uk/~susan/,rD32w_EAAAAJ
+Edwin R. Hancock,University of York,https://www-users.cs.york.ac.uk/~erh/,EjDU2ncAAAAJ
+Adrian G. Bors,University of York,https://www-users.cs.york.ac.uk/~adrian/,cvdyalUAAAAJ
+Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det/,nh6AHbIAAAAJ
+Nicholas Drivalos Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos/,Dvp7fNoAAAAJ
+John A McDermid,University of York,https://www-users.cs.york.ac.uk/~jam/,REscLkoAAAAJ
+William A. P. Smith,University of York,https://www-users.cs.york.ac.uk/wsmith/,o2z0h6MAAAAJ
+Anders Drachen,University of York,https://www.cs.york.ac.uk/people/?group=Academic%20and%20Teaching%20Staff&username=adrachen,rOcL0NgAAAAJ
+Samuel L. Braunstein,University of York,https://www-users.cs.york.ac.uk/~schmuel/,jh-mlXoAAAAJ
+Angus M. Marshall,University of York,https://www-users.york.ac.uk/~amm616/,RV4F9n8AAAAJ
+Chris Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb/,QQ_dWtcAAAAJ
+Steven A. Wright,University of York,https://www-users.cs.york.ac.uk/~saw/,uh1UaD4AAAAJ
+Dimitrios S. Kolovos,University of York,https://www-users.cs.york.ac.uk/~dkolovos/,aRd1i6UAAAAJ
+James Cussens,University of York,https://www-users.cs.york.ac.uk/~jc/,OEFkOR0AAAAJ
+Steve King,University of York,https://www-users.cs.york.ac.uk/~king/,Q75-BIgAAAAJ
+Rob Alexander,University of York,https://www-users.cs.york.ac.uk/~rda/,DFdBpC0AAAAJ
+Richard C. Wilson,University of York,https://www-users.cs.york.ac.uk/~wilson/,o4Sq5yMAAAAJ
+Nick Pears,University of York,https://www-users.cs.york.ac.uk/nep/,U_hUs6wzw_AC
+Vassilios G. Vassilakis,University of York,https://www-users.cs.york.ac.uk/~vv/,-OzxgUwAAAAJ
+Siyuan Ji,University of York,https://www.cs.york.ac.uk/people/?group=Academic%20and%20Teaching%20Staff&username=ji,oZdrNNwAAAAJ
+Jim Woodcock,University of York,https://www-users.cs.york.ac.uk/~jim/,bO-FeTIAAAAJ
+Peter Nightingale,University of York,https://www-users.cs.york.ac.uk/pwn503/,Msh14AUAAAAJ
+Ioanna Iacovides,University of York,https://www-users.york.ac.uk/~ii560/,LuBaasYAAAAJ
+Richard F. Paige,University of York,https://www-users.cs.york.ac.uk/~paige/,HzFjuyUAAAAJ
+Peter I. Cowling,University of York,https://www-users.cs.york.ac.uk/~pic/,xQ7IpL8AAAAJ
+Delaram Kahrobaei,University of York,https://www-users.cs.york.ac.uk/~delaram/,ScTS9PkAAAAJ
+Jim Austin,University of York,https://www-users.cs.york.ac.uk/~austin/,xpZL8d8AAAAJ
+James Alfred Walker,University of York,https://www.cs.york.ac.uk/people/?group=Research%20Fellows%20and%20Research%20Associates&username=jawalker,Yl5OycsAAAAJ
diff --git a/dblp-aliases.csv b/dblp-aliases.csv
index 6e4ade2e..38c9bbe5 100644
--- a/dblp-aliases.csv
+++ b/dblp-aliases.csv
@@ -32637,4 +32637,6 @@ Torsten Kroeger,Torsten Kröger
 Alexander H. Waibel,Alex Waibel
 Walter Tichy,Walter F. Tichy
 Anne Martens,Anne Koziolek
-David Mimno,David M. Mimno
\ No newline at end of file
+David Mimno,David M. Mimno
+Chris Bailey,Christopher Bailey 0002
+Jim Austin,James Austin

emeryberger commented on csrankings.csv:
```diff
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
+Edwin R. Hancock,University of York,https://www-users.cs.york.ac.uk/~erh/,EjDU2ncAAAAJ
+Adrian G. Bors,University of York,https://www-users.cs.york.ac.uk/~adrian/,cvdyalUAAAAJ
+Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det/,nh6AHbIAAAAJ
+Nicholas Drivalos Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos/,Dvp7fNoAAAAJ
```
Actual URL is https://nmatra.github.io/

iangray001 replied:
No, this is the correct address. Nikos may move his hosting at some point but whilst he is a member of the department this is his web address.

emeryberger commented on csrankings.csv:
```diff
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
+Steven A. Wright,University of York,https://www-users.cs.york.ac.uk/~saw/,uh1UaD4AAAAJ
+Dimitrios S. Kolovos,University of York,https://www-users.cs.york.ac.uk/~dkolovos/,aRd1i6UAAAAJ
+James Cussens,University of York,https://www-users.cs.york.ac.uk/~jc/,OEFkOR0AAAAJ
+Steve King,University of York,https://www-users.cs.york.ac.uk/~king/,Q75-BIgAAAAJ
```
Needs to be "Steve King 0001". See https://dblp.uni-trier.de/pers/hd/k/King_0001:Steve

emeryberger commented on csrankings.csv:
```diff
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
+Adrian G. Bors,University of York,https://www-users.cs.york.ac.uk/~adrian/,cvdyalUAAAAJ
+Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det/,nh6AHbIAAAAJ
+Nicholas Drivalos Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos/,Dvp7fNoAAAAJ
+John A McDermid,University of York,https://www-users.cs.york.ac.uk/~jam/,REscLkoAAAAJ
```
Missing period after middle initial.

emeryberger commented on csrankings.csv:
```diff
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
+Wanli Chang,University of York,https://www-users.cs.york.ac.uk/~wanli/,OYcT_B4AAAAJ
+Andy J. Wellings,University of York,https://www-users.cs.york.ac.uk/~andy/,yPa4QrkAAAAJ
+Neil C. Audsley,University of York,https://www-users.cs.york.ac.uk/~neil/,D65ZWSgAAAAJ
+Ian Gray,University of York,https://www-users.cs.york.ac.uk/~iang/,HloK4CAAAAAJ
```
Web page should be https://iangray.me.uk/

iangray001 replied:
No, this is the correct address. I may move my hosting at some point but whilst I am a member of the department this is my web address.

iangray001 committed:
Updated King and McDermid's names
diff --git a/csrankings.csv b/csrankings.csv
index e0d98d14..056ec73e 100644
--- a/csrankings.csv
+++ b/csrankings.csv
@@ -15817,7 +15817,7 @@ Edwin R. Hancock,University of York,https://www-users.cs.york.ac.uk/~erh/,EjDU2n
 Adrian G. Bors,University of York,https://www-users.cs.york.ac.uk/~adrian/,cvdyalUAAAAJ
 Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det/,nh6AHbIAAAAJ
 Nicholas Drivalos Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos/,Dvp7fNoAAAAJ
-John A McDermid,University of York,https://www-users.cs.york.ac.uk/~jam/,REscLkoAAAAJ
+John A. McDermid,University of York,https://www-users.cs.york.ac.uk/~jam/,REscLkoAAAAJ
 William A. P. Smith,University of York,https://www-users.cs.york.ac.uk/wsmith/,o2z0h6MAAAAJ
 Anders Drachen,University of York,https://www.cs.york.ac.uk/people/?group=Academic%20and%20Teaching%20Staff&username=adrachen,rOcL0NgAAAAJ
 Samuel L. Braunstein,University of York,https://www-users.cs.york.ac.uk/~schmuel/,jh-mlXoAAAAJ
@@ -15826,7 +15826,7 @@ Chris Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb/,QQ_dWtc
 Steven A. Wright,University of York,https://www-users.cs.york.ac.uk/~saw/,uh1UaD4AAAAJ
 Dimitrios S. Kolovos,University of York,https://www-users.cs.york.ac.uk/~dkolovos/,aRd1i6UAAAAJ
 James Cussens,University of York,https://www-users.cs.york.ac.uk/~jc/,OEFkOR0AAAAJ
-Steve King,University of York,https://www-users.cs.york.ac.uk/~king/,Q75-BIgAAAAJ
+Steve King 0001,University of York,https://www-users.cs.york.ac.uk/~king/,Q75-BIgAAAAJ
 Rob Alexander,University of York,https://www-users.cs.york.ac.uk/~rda/,DFdBpC0AAAAJ
 Richard C. Wilson,University of York,https://www-users.cs.york.ac.uk/~wilson/,o4Sq5yMAAAAJ
 Nick Pears,University of York,https://www-users.cs.york.ac.uk/nep/,U_hUs6wzw_AC
